src/main/java/org/fundacionjala/coding/escarleth/movies/MovieNewRelease.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.escarleth.movies;
+
+/**
+ * Powered by Escarleth L.
+ */
+public class MovieNewRelease extends Movie {
+    private static final int AMOUNT = 3;
+
+    /**
+     * Constructor for MovieNewRelease.
+     *
+     * @param title of the Movie.
+     */
+    public MovieNewRelease(String title) {
+        super(title);
+    }
+
+    /** determineRentalAmount method.
+     * {@inheritDoc}
+     */
+    @Override
+    double determineRentalAmount(int daysRented) {
+        return daysRented * AMOUNT;
+    }
+
+    /**
+     * @param daysRented is the quantity of rental days.
+     * @return the total frequent renter points .
+     */
+    int determinateFrequentRenterPoints(int daysRented) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)